ts/indicators/volume-by-price.src.ts

```diff
@@ -361,10 +361,16 @@ seriesType<Highcharts.VBPIndicator>(
             init: boolean
         ): void {
             var series = this,
-                attr: Highcharts.SVGAttributes = {};
+                inverted = series.chart.inverted,
+                attr: Highcharts.SVGAttributes = {},
+                translate,
+                position;
 
             if (!init) {
-                attr.translateX = series.yAxis.pos;
+                translate = inverted ? 'translateY' : 'translateX';
+                position = inverted ? series.yAxis.top : series.xAxis.left;
+                (series.group as any)['forceAnimate:' + translate] = true;
```

Comment:
What is the reason of using `series.group as any` instead of `SVGElement` ?

Reply:
Corrected. Thanks for noticing!